rpc/proto/mayastor.proto

```diff
@@ -273,6 +277,28 @@ message UnpublishNexusRequest {
   string uuid = 1;   // uuid of the nexus which to destroy
 }
 
+enum NvmeAnaState {
+  NVME_ANA_INVALID_STATE = 0;           // invalid, do not use
+  NVME_ANA_OPTIMIZED_STATE = 0x1;
+  NVME_ANA_NON_OPTIMIZED_STATE = 0x2;
+  NVME_ANA_INACCESSIBLE_STATE = 0x3;
+  NVME_ANA_PERSISTENT_LOSS_STATE = 0x4; // not yet supported
+  NVME_ANA_CHANGE_STATE = 0xF;          // not yet supported
+}
```

Comment:
I get what optimized and non-optimized states can be used for. What is the meaning and typical use case for inaccessible, persistent loss and change states?

Reply:
- Inaccessible: Commands processed by a controller that reports this state for an ANA Group are not able to access user data of namespaces in that ANA Group. 
- Persistent loss:  Commands processed by a controller that reports this state for an ANA Group are persistently not able to access user data of namespaces in that ANA Group. The relationship between a controller and an ANA Group in this state shall not transition to any other ANA state.
- Change state: The change from one asymmetric namespace access state to another asymmetric namespace access state is called a transition. Transitions may occur in such a way that the ANA Change state is not visible to the
host (i.e., the ANA Change state may or may not be reported in the Asymmetric Namespace Access State field in the Asymmetric Namespace Access log).